src/logging/request-obfuscator.js

```diff
@@ -0,0 +1,101 @@
+
+/**
+ * Module dependencies.
+ */
+
+import _ from 'lodash';
+import methods from '../methods';
+
+/**
+ * Get method by its lowerCase.
+ */
+
+function getMethod(method) {
+  return methods[_.findKey(methods, (object, key) => {
```

Comment:
Pretty expensive to run this on every request. Can you cache the lowercase mapping using _.mapKeys instead?